samples/startSample.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ "$1" == "-h" ] || [ $# -lt 1 ]; then
+  echo "Usage: `basename $0` scriptname [OPTIONS] "
+  exit 0
+fi
+
+# Will run a sample without requiring an install of the library.
+PYTHONPATH="$(dirname $( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd ))"
+export PYTHONPATH=$PYTHONPATH
+python "$@"
```

Comment:
I don't like how unix-specific this is. I also don't like the idea of requiring a bootstrapping shell script in general. Can you please just dynamically alter the `PYTHONPATH` like so:

```python
import sys
sys.path.insert(0, "../") # add ../ to PYTHONPATH
from matrix_client.client import MatrixClient
from matrix_client.api import MatrixRequestError
```